content/posts/downstream-automation.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Downstream automation is here"
+date: 2022-03-02
+weight: 68
+---
+
+# Downstream automation is here
+
+Finally, it's here. Now, you can do the whole Fedora release with the help of Packit.
+Let's take a look at how this works on an example of [OGR](https://github.com/packit/ogr), the Python library we develop:
+
+## Upstream
+
+The process of releasing a new version starts in the upstream repository.
+Here, we can see an upstream release:
+
+TODO: upstream release image
+
+## Propose downstream
+
+As the first step on our way to Fedora users, we need to get the new upstream release to the Fedora dist-git.
+This is what we call `propose-downstream` job.
+Here is a snippet from the config file of OGR:
+
+```yaml
+downstream_package_name: python-ogr
+copy_upstream_release_description: true
+
+jobs:
+  - job: propose_downstream
+    trigger: release
+    metadata:
+      dist_git_branches:
+        - fedora-all
+        - epel-8
+```
+
+And how does this work?
+Packit gets the information about the newly created release from GitHub (via webhook).
+load the config from the release commit and if there is a `propose-downstream` job
+defined, the workflow begins.
+As a first step, the archive is saved to lookaside cache
+and after that, Packit updates the dist-git content (mainly `sources` file and spec-file)
+via pull-requests for the specified branches. (Direct push is possible by setting a
+[`create_pr` option](https://packit.dev/docs/configuration/#create_pr) to `false`.)
```

Comment:
Could we split this paragraph into two sections?
1. What work is happening behind the scenes (clone & checkout, update spec file & %changelog, push, sources, create PRs)
2. The implementation details - how it works

Reply:
Good point. PTAL

Reply:
Thanks Franto! Nicely done